lib/screens/groups/widgets/group_page_body.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:day2day/routes.dart';
+import 'package:day2day/screens/groups/bloc/groups_bloc.dart';
+import 'package:day2day/services/groups_service/groups_service.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class BottomSheetContent extends StatefulWidget {
+  @override
+  __BottomSheetContentState createState() => __BottomSheetContentState();
+}
+
+class __BottomSheetContentState extends State<BottomSheetContent> {
+  GroupsBloc _groupsBloc;
+  final GroupsRepository groupsRepository = GroupsRepository();
+
+  @override
+  void initState() {
+    super.initState();
+    _groupsBloc = GroupsBloc(groupsRepository: GroupsRepository());
+    _groupsBloc.add(FetchGroups());
+  }
+
+  @override
+  void dispose() {
```

Comment:
`BlocProvider` will close the bloc so we don't need to manually call it

Reply:
Oh, I forgot